phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/LeftToggleGroup.java

```diff
@@ -86,6 +89,19 @@ private LeftToggleGroup(String name, Supplier<ToggleButton> allButtonSupplier,
         Bindings.bindContent(this.toggleGroup.getToggles(), this.adhocToggleButtons);
         Bindings.bindContent(this.toggleButtonBox.getChildren(), this.adhocToggleButtons);
 
+        this.adhocToggleButtons.addListener(new ListChangeListener<ToggleButton>() {
```

Comment:
This listener is required to ensure that one `ToggleButton` is always selected.
Let's assume for example you've selected the category `Games` and enter `note`, then you'll only have the two categories All and Development tools (or something) left, of which no category is selected, this would be strange.